src/Service/NotificationReceiverService.php

```diff
@@ -114,7 +114,7 @@ public function process(Request $requestObject)
         }
 
         // Is the plugin configured to live environment
-        $pluginMode = $this->configurationService->getEnvironment();
+        $pluginMode = $this->configurationService->getEnvironment(null);
```

Comment:
shouldn't we define a default value instead of passing null as a parameter?

Reply:
Passing null returns the config value for the default sales channel, do you mean we should hardcode `live` or `test` for notification receiving?

Reply:
what I meant if we don't pass null but nothing then it should retrieve the default sales channel: `getEnvironment()` and if we know the salesChannelId then: `getEnvironment($salesChannelId)`

Reply:
Gotcha, made the params nullable now